{"uid":"86adf0632bd0476e","name":"Login with invalid credentials test","fullName":"tests.test_login.TestLogin#test_invalid_login","historyId":"d4a74b5adfb086f235dc50992a7adb79","time":{"start":1704071418868,"stop":1704071429074,"duration":10206},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9c0dd6d33 <unknown>\n#1 0x55d9c0a93f87 <unknown>\n#2 0x55d9c0ae3c0e <unknown>\n#3 0x55d9c0ae3d11 <unknown>\n#4 0x55d9c0b2f004 <unknown>\n#5 0x55d9c0b0d36d <unknown>\n#6 0x55d9c0b2c297 <unknown>\n#7 0x55d9c0b0d0e3 <unknown>\n#8 0x55d9c0ad5044 <unknown>\n#9 0x55d9c0ad644e <unknown>\n#10 0x55d9c0d9b861 <unknown>\n#11 0x55d9c0d9f785 <unknown>\n#12 0x55d9c0d89285 <unknown>\n#13 0x55d9c0da041f <unknown>\n#14 0x55d9c0d6d20f <unknown>\n#15 0x55d9c0dc4028 <unknown>\n#16 0x55d9c0dc41f7 <unknown>\n#17 0x55d9c0dd5ed4 <unknown>\n#18 0x7f4109294ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f7244af1150>\nemail = 'nirt236@gmail.com', password = '123456'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"43eb2bd234229a61fae4dc71ac43fa61\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f72448d2c00>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55d9c0dd6d33 <unknown>\nE       #1 0x55d9c0a93f87 <unknown>\nE       #2 0x55d9c0ae3c0e <unknown>\nE       #3 0x55d9c0ae3d11 <unknown>\nE       #4 0x55d9c0b2f004 <unknown>\nE       #5 0x55d9c0b0d36d <unknown>\nE       #6 0x55d9c0b2c297 <unknown>\nE       #7 0x55d9c0b0d0e3 <unknown>\nE       #8 0x55d9c0ad5044 <unknown>\nE       #9 0x55d9c0ad644e <unknown>\nE       #10 0x55d9c0d9b861 <unknown>\nE       #11 0x55d9c0d9f785 <unknown>\nE       #12 0x55d9c0d89285 <unknown>\nE       #13 0x55d9c0da041f <unknown>\nE       #14 0x55d9c0d6d20f <unknown>\nE       #15 0x55d9c0dc4028 <unknown>\nE       #16 0x55d9c0dc41f7 <unknown>\nE       #17 0x55d9c0dd5ed4 <unknown>\nE       #18 0x7f4109294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1704071368606,"stop":1704071368606,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1704071368606,"stop":1704071368606,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1704071368607,"stop":1704071368607,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9c0dd6d33 <unknown>\n#1 0x55d9c0a93f87 <unknown>\n#2 0x55d9c0ae3c0e <unknown>\n#3 0x55d9c0ae3d11 <unknown>\n#4 0x55d9c0b2f004 <unknown>\n#5 0x55d9c0b0d36d <unknown>\n#6 0x55d9c0b2c297 <unknown>\n#7 0x55d9c0b0d0e3 <unknown>\n#8 0x55d9c0ad5044 <unknown>\n#9 0x55d9c0ad644e <unknown>\n#10 0x55d9c0d9b861 <unknown>\n#11 0x55d9c0d9f785 <unknown>\n#12 0x55d9c0d89285 <unknown>\n#13 0x55d9c0da041f <unknown>\n#14 0x55d9c0d6d20f <unknown>\n#15 0x55d9c0dc4028 <unknown>\n#16 0x55d9c0dc41f7 <unknown>\n#17 0x55d9c0dd5ed4 <unknown>\n#18 0x7f4109294ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f7244af1150>\nemail = 'nirt236@gmail.com', password = '123456'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"43eb2bd234229a61fae4dc71ac43fa61\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f72448d2c00>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55d9c0dd6d33 <unknown>\nE       #1 0x55d9c0a93f87 <unknown>\nE       #2 0x55d9c0ae3c0e <unknown>\nE       #3 0x55d9c0ae3d11 <unknown>\nE       #4 0x55d9c0b2f004 <unknown>\nE       #5 0x55d9c0b0d36d <unknown>\nE       #6 0x55d9c0b2c297 <unknown>\nE       #7 0x55d9c0b0d0e3 <unknown>\nE       #8 0x55d9c0ad5044 <unknown>\nE       #9 0x55d9c0ad644e <unknown>\nE       #10 0x55d9c0d9b861 <unknown>\nE       #11 0x55d9c0d9f785 <unknown>\nE       #12 0x55d9c0d89285 <unknown>\nE       #13 0x55d9c0da041f <unknown>\nE       #14 0x55d9c0d6d20f <unknown>\nE       #15 0x55d9c0dc4028 <unknown>\nE       #16 0x55d9c0dc41f7 <unknown>\nE       #17 0x55d9c0dd5ed4 <unknown>\nE       #18 0x7f4109294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1704071418868,"stop":1704071429073,"duration":10205},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9c0dd6d33 <unknown>\n#1 0x55d9c0a93f87 <unknown>\n#2 0x55d9c0ae3c0e <unknown>\n#3 0x55d9c0ae3d11 <unknown>\n#4 0x55d9c0b2f004 <unknown>\n#5 0x55d9c0b0d36d <unknown>\n#6 0x55d9c0b2c297 <unknown>\n#7 0x55d9c0b0d0e3 <unknown>\n#8 0x55d9c0ad5044 <unknown>\n#9 0x55d9c0ad644e <unknown>\n#10 0x55d9c0d9b861 <unknown>\n#11 0x55d9c0d9f785 <unknown>\n#12 0x55d9c0d89285 <unknown>\n#13 0x55d9c0da041f <unknown>\n#14 0x55d9c0d6d20f <unknown>\n#15 0x55d9c0dc4028 <unknown>\n#16 0x55d9c0dc41f7 <unknown>\n#17 0x55d9c0dd5ed4 <unknown>\n#18 0x7f4109294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"4dd4ee12d83e5dea","name":"public ip address","source":"4dd4ee12d83e5dea.txt","type":"text/plain","size":13},{"uid":"25af7ea447c3ffff","name":"Cookies","source":"25af7ea447c3ffff.json","type":"application/json","size":2},{"uid":"261d594e2fd0403a","name":"Session Storage","source":"261d594e2fd0403a.json","type":"application/json","size":2},{"uid":"fd79a0af758ce79a","name":"Local Storage","source":"fd79a0af758ce79a.json","type":"application/json","size":2},{"uid":"d3d5f96fe5ce4762","name":"Console Logs","source":"d3d5f96fe5ce4762.json","type":"application/json","size":2},{"uid":"2473461f94fad6e8","name":"Full Page Screenshot","source":"2473461f94fad6e8.png","type":"image/png","size":2655641},{"uid":"618eeaaab7566b02","name":"URL","source":"618eeaaab7566b02.uri","type":"text/uri-list","size":25},{"uid":"3374e536b3fe7e8f","name":"Network Logs","source":"3374e536b3fe7e8f.json","type":"application/json","size":13980}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az695-453"},{"name":"thread","value":"2278-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'nirt236@gmail.com'"},{"name":"password","value":"'123456'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"86adf0632bd0476e.json","parameterValues":["'nirt236@gmail.com'","'123456'"]}